masterlist.yaml

```diff
@@ -12615,6 +12613,9 @@ plugins:
         condition: 'active("Convenient Horses.esp") and not active("CH (GrayFox_)?(Bruma_)?(Shire_)?(BeyondReach_)?(Falskaar_)?(Vigilant_)?Wyrm.*\.esp") and not active("CH (GFC_)?(BRU_)?(SHI_)?(BRE_)?(FAL_)?(VIG_)?WYR.*\.esp")'
       - <<: *patch3rdParty_KPH_CACO
         condition: 'active("Complete Alchemy & Cooking Overhaul.esp") and not active("CACO_Wyrmstooth_Patch.esp")'
+      - <<: *obsolete
+        subs: [ '[Wyrmstooth](https://www.nexusmods.com/skyrimspecialedition/mods/45565/)' ]
+        condition: checksum("Wyrmstooth.esp", FD63BB73) or checksum("Wyrmstooth.esp", 85D4BF27) or checksum("Wyrmstooth.esp", 97F8C8B4) or checksum("Wyrmstooth.esp", F73A6161) or version("Wyrmstooth.esp", "1.19", <) 
```

Comment:
The plugin on the nexus includes version info in its header description.

![image](https://user-images.githubusercontent.com/1944639/107792121-b06fef80-6d4c-11eb-8422-ef60855b7775.png)

You can use `not version("Wyrmstooth.esp", "1.19", >=)` instead of checking for each old CRC.

Also the condition needs to be wrapped with ' '.
condition: 'checksum("Wyrmstooth.esp", FD63BB73) and ...'